services/vod-to-live/service/src/index.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  CreateRabbitMQConnectivityMetric,
+  envelopeLoggingMiddleware,
+  setupMessagingBroker,
+} from '@axinom/mosaic-message-bus';
+import {
+  closeHttpServer,
+  handleGlobalErrors,
+  Logger,
+  MosaicErrors,
+  setupGlobalConsoleOverride,
+  setupGlobalLogMiddleware,
+  setupGlobalSkipMaskMiddleware,
+  setupLivenessAndReadiness,
+  setupMonitoring,
+  setupShutdownActions,
+  tenantEnvironmentIdsLogMiddleware,
+  trimErrorsSkipMaskMiddleware,
+} from '@axinom/mosaic-service-common';
+import bodyParser from 'body-parser';
+import express from 'express';
+import { getFullConfig } from './common';
+import {
+  AzureStorage,
+  KeyServiceApi,
+  setupPrePublishingValidationWebhook,
+  VirtualChannelApi,
+} from './domains';
+import { registerMessaging } from './messaging/register-messaging';
+
+const logger = new Logger({ context: 'bootstrap' });
+
+// Entry point for the service. For annotated version please see /services/media/service/src/index.ts.
+async function bootstrap(): Promise<void> {
+  handleGlobalErrors(logger);
+  setupGlobalSkipMaskMiddleware(trimErrorsSkipMaskMiddleware);
+  setupGlobalConsoleOverride(logger);
+  const app = express();
+  app.use(
+    bodyParser.json({
+      limit: '50mb',
+    }),
+  );
+  const config = getFullConfig();
+  setupGlobalLogMiddleware([tenantEnvironmentIdsLogMiddleware(config)]);
+
+  const { readiness } = setupLivenessAndReadiness(config);
+
+  const shutdownActions = setupShutdownActions(app, logger);
+
+  const storage = new AzureStorage(
+    config.azureStorageConnection,
+    config.azureBlobContainerName,
+  );
+
+  const virtualChannelApi = new VirtualChannelApi(
+    config.virtualChannelManagementApiBaseUrl,
+    config.virtualChannelManagementApiKey,
+  );
+
+  const keyServiceApi = new KeyServiceApi(config);
+  const broker = await setupMessagingBroker({
```

Comment:
This service has no database, so we cannot use the postgres counter we use in other services to support consistent message re-delivery policy. @Zehelein any thoughts?

Reply:
Discussed, acceptable for now. Adding a database just for this case is too much.